plugin.video.mytv_bg/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.mytv_bg" name="MyTV.BG" version="2.0.4" provider-name="jordan@mytv.bg">
+  <requires>
+    <import addon="xbmc.python" version="2.1.1"/>
```

Comment:
please set xmbc.python version to 2.24.0  so it matches jarvis: http://kodi.wiki/view/addon.xml